src/axolotl/train.py

```diff
@@ -495,6 +496,10 @@ def train(
         processor,
     ) = setup_model_and_trainer(cfg, dataset_meta)
 
+    plugin_manager = PluginManager.get_instance()
+    plugin_manager.post_model_load(cfg, model)
+    plugin_manager.post_lora_load(cfg, model)
```

Comment:
Seems like these two have no differences? Should `.post_model_load` be run prior to loading any lora adapters?

Reply:
Agreed with @NanoCode012!